lichess-bot.py

```diff
@@ -289,7 +290,7 @@ def play_game(li, game_id, control_queue, engine_factory, user_profile, config,
                             best_move = choose_move_time(engine, board, correspondence_move_time, can_ponder, draw_offered)
                         else:
                             best_move = choose_move(engine, board, game, can_ponder, draw_offered, start_time, move_overhead)
-                    if best_move.resigned:
+                    if best_move.resigned and len(board.move_stack) >= 2:
```

Comment:
What is the reason for this change?

Reply:
I added `and len(board.move_stack) >= 2` because if the bot resigns on the first move, the game is instead aborted.